programs/voter-stake-registry/tests/test_all_deposits.rs

```diff
@@ -9,7 +9,7 @@ use voter_stake_registry::state::LockupKind;
 mod program_test;
 
 #[allow(unaligned_references)]
-#[tokio::test]
+//#[tokio::test]ß
```

Comment:
looks like a typo

Reply:
Yeah, I was troubleshooting the CI test failures